client/src/game/ui/reactive/view/cardsView.ts

```diff
@@ -110,6 +110,23 @@ const subscribeToCardChanges = (order: number) => {
     location: c.location,
   }), () => updateBorder(order));
 
+  // Card fade and critical indicator
+  subFullState((s) => {
+    const card = s.visibleState!.deck[order];
+    const status = (card.suitIndex !== null && card.rank !== null)
+      ? s.visibleState!.cardStatus[card.suitIndex!][card.rank!]
```

Comment:
why put the bangs on suitindex and rank?

Reply:
I didn't know how to address the nullability besides this approach. I'm new to TS so any suggestion welcome

Reply:
Wow I guess I can just remove them. What kind of sorcery is typescript doing??